src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid.Tests/Properties/AssemblyInfo.cs

```diff
@@ -17,6 +17,6 @@
 
 [assembly: ComVisible(false)]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
The change in this file should be reverted - this is the dataplane AssemblyInfo.

Reply:
fixed